server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -595,11 +594,6 @@ private QueryDataSet processShowDynamicParameterQuery() {
         timestamp++,
         "storage group number",
         Integer.toString(IoTDB.metaManager.getAllStorageGroupNames().size()));
-    addRowRecordForShowQuery(
```

Comment:
the show dynamic parameter statement could be deleted. The statistics could be queried by other statements

Reply:
Deleted